apps/condo/domains/meter/integrations/sendVerificationDateReminder.js

```diff
@@ -0,0 +1,63 @@
+const dayjs = require('dayjs')
```

Comment:
Why do you put the logic here? It's not an external integration. It's typical business logic. Put it in the tasks. And if your need some server-side utils put them to the `utils/serverSchema`

Reply:
I just put into the separate file, since I would like to keep definition of tasks and real task implementation in separate files

Reply:
please, move this script to task folder
```
@condo/domains/resident/tasks or @condo/domains/meter/tasks
```
we use integration folder only for external integrations